src/operator/context/account.tsx

```diff
@@ -168,17 +180,43 @@ export const AccountProvider: FC<Props> = React.memo(({children}) => {
     }
   }, [dispatch, history, accountID])
 
+  const handleMigrateOrgs = useCallback(
+    async (toAccountId: string) => {
+      try {
+        setMigrateStatus(RemoteDataState.Loading)
+        const resp = await patchOperatorAccountsMigrate({
+          fromAccountId: accountID,
+          toAccountId: toAccountId,
+        })
+
+        if (resp.status !== 200) {
+          throw new Error(resp.data.message)
+        }
+
+        setMigrateOverlayVisible(false)
+        setMigrateStatus(RemoteDataState.Done)
```

Comment:
I might reverse the order of lns `196` and `197` here, since `migrateStatus` is used in the migration overlay. By the time `migrateStatus` is `Done` I suspect the overlay will be removed from the DOM.

Reply:
That's a good point. Changing it.